src/transformers/models/switch_transformers/modeling_switch_transformers.py

```diff
@@ -1865,4 +1865,4 @@ def forward(
             return_dict=return_dict,
         )
 
-        return encoder_outputs
+        return encoder_outputs
```

Comment:
New line at the end of the file needed. This will resolve the failing code quality checks.

```suggestion
        return encoder_outputs

```